sentry/src/main/java/io/sentry/SentryUUID.java

```diff
@@ -0,0 +1,299 @@
+package io.sentry;
+
+import java.util.Arrays;
+import java.util.Random;
+import java.util.UUID;
+
+/**
+ * Utility class for faster id generation for SentryId, SpanId, and unique filenames uses throughout
+ * the SDK It uses our vendored Random class instead of SecureRandom for improved performance It
```

Comment:
But it doesn't 😅

Reply:
not yet :)
Sorry, wanted to add a comment here in the PR, that we need to first merge the `main` branch to 8.x.x and then to this one